docs/TESTING.md

```diff
@@ -0,0 +1,150 @@
+---
+For release: v1.0
+---
+
+# Test Plan
+
+## Introduction
+
+The idea of this The Test Plan is to communicate the test approach to team members. 
+
+As the team works with the product, they will define the needs for the vNext.
+
+### Team communication
+
+Perform communication through Slack and Github issues.
```

Comment:
Could you expand on this?

Reply:
Not sure what to say more, I just wanted to make sure that everyone testing knows that we communicate on Slack and Github. Do you recommend to delete this section at all?